src/ServiceControl.LoadTests.AuditGenerator/Program.cs

```diff
@@ -1,115 +1,397 @@
+                {
+                    Dispatcher = dispatchMessages;
+                }
 
-            var minLength = SettingsReader<int>.Read(ConfigRoot, "MinLength", 10000);
-            var maxLength = SettingsReader<int>.Read(ConfigRoot, "MaxLength", 20000);
-            var endpointName = SettingsReader<string>.Read(ConfigRoot, "EndpointName", "AuditGen");
-            var connectionString = SettingsReader<string>.Read(ConfigRoot, "TransportConnectionString", "");
+                protected override Task OnStart(IMessageSession session)
+                {
+                    return Task.CompletedTask;
+                }
 
-            HostId = Guid.NewGuid().ToString("N");
+                protected override Task OnStop(IMessageSession session)
+                {
+                    return Task.CompletedTask;
+                }
+            }
+        }
 
-            var config = new EndpointConfiguration(endpointName);
-            config.AssemblyScanner().ExcludeAssemblies("ServiceControl");
-            config.UseSerialization<NewtonsoftJsonSerializer>();
+        static void GenerateLayouts()
+        {
+            var random = new Random();
+            for (var i = 0; i < numberOfLayouts; i++)
+            {
+                var messageTypeName = RandomString(MessageTypeNameAndValueLength, random).FirstCharToUpper();
+                var messageTypeNameLength = Encoding.UTF8.GetBytes(messageTypeName).Length;
+                var layoutBuilder = new StringBuilder();
+                layoutBuilder.AppendLine();
+                var propertyPositionPlaceHolder = 0;
+                for (var j = 1; j < bodySize; j++)
+                {
+                    var propertyName = RandomString(random.Next(10, 30), random).FirstCharToUpper();
+                    layoutBuilder.AppendLine(
+                        $"   <{propertyName}>{{{propertyPositionPlaceHolder}}}</{propertyName}>");
+                    propertyPositionPlaceHolder++;
 
-            var customization = (TransportCustomization)Activator.CreateInstance(Type.GetType(customizationTypeName, true));
-            var transportSettings = new TransportSettings
+                    var intermediateLayout = MessageBaseLayout.Replace("__MESSAGETYPENAME__", messageTypeName)
+                        .Replace("__CONTENT__", layoutBuilder.ToString());
+                    // saving 20 chars per property
+                    if ((Encoding.UTF8.GetBytes(intermediateLayout).Length + (i * messageTypeNameLength)) >= bodySize)
+                    {
+                        MessageBaseLayouts.Add((intermediateLayout, propertyPositionPlaceHolder));
+                        break;
+                    }
+                }
+            }
+        }
+
+        public static string RandomString(int length, Random random)
+        {
+            const string chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
```

Comment:
I guess it could become a problem for future features in SC that parses the payloads in some way

@danielmarbach do U think it would have a negative impact if we just removed the `0123456789` part?